app/views/requirements/show.html.erb

```diff
@@ -0,0 +1,35 @@
+<% content_for :sidebar do %>
+  <h2>Requirement</h2>
+  <%= sidebar_button_link 'Return to Task', @requirement.task %>
+  <%= sidebar_button_link 'Return to Requirements List', requirements_path %>
+  <%= sidebar_button_link 'Edit Requirement', edit_requirement_path(@requirement) if can? :edit, @requirement %>
+<% end %>
+
+Display option 1:
+<p>
+  <span class="h3"><%= @requirement.skill %> for <%= link_to @requirement.task.title, task_path(@requirement.task) %></span>
+  (<%= link_to @requirement.task.event.title, event_path(@requirement.task.event) %> on <%= format_date_value(@requirement.task.start_time) %>)
+</p>
+
+Display option 2:
+<p>
+  <span class="h3"><%= @requirement.skill %></span> for
+  <span class="h4"><%= link_to @requirement.task.event.title, event_path(@requirement.task.event) %></span>:
+  <span class="h5"><%= link_to @requirement.task.title, task_path(@requirement.task)%></span>
+  <% if @requirement.task.event.location.present? %>
```

Comment:
That's a lot of dots...
We should change this to be a method on task. The task location should be either a tasks's location, if defined, or the event's location.